admin/src/css/configuration.less

```diff
@@ -92,3 +92,80 @@ dl.horizontal {
     font-size: 100%;
   }
 }
+.message-queue {
+  .flex-row {
```

Comment:
I think `.flex-row` is a concept we'll want to reuse so don't constrain it to `.message-queue` to make it easier.